### DISCUSSION TITLE
ACME provisioner added but not listed

### DISCUSSION THREAD

**Discussion by kodakrome** [smallstep/certificates#2036] [Q&A]
everything seemed to be going well until I tried to configure ACME
I see ACME as the only listed provisioner in my `ca.json` 
I've restarted multiple times, but no matter what I can only see a JWK provisioner and I don't know why
`step ca provisioner list` shows 'JWK'
curl https://smallstep/acme/acme/ (and many different iterations) gives 404

Any help greatly appreciated, I really am trying here...

**Comment by tashian** [comment]
Did you enable remote provisioner management? It sounds like you may have.
If so, then your provisioner configurations will be stored in the database, and will manage your provisioners via `step ca provisioner [add|update]`, not by editing `ca.json`.
(And, the items in `ca.json` will be ignored.)
